crates/rpc/types/src/types.rs

```diff
@@ -164,6 +163,17 @@ pub struct RpcBlockHeader {
     pub state_root: [u8; 32],
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RpcBlockBody {
+    /// The fields of block body is to be decided.
+    pub todo: u64,
+}
```

Comment:
We don't want to directly expose the block bodies as json since that makes consumers dependent on the json representation which we don't want to directly expose.  If we do want to expose it for testing purposes, we should make it under an explicitly-unstable `debug_` namespace.